components/Header.js

```diff
@@ -4,24 +4,25 @@ import styled from "styled-components";
 function Header() {
   return (
     <Link href={`/`}>
-      <Logo>/ˈsnɪp.ɪt/</Logo>
+      <StyledHeader aria-label="Snip it Logo">
+        <StyledH1 aria-hidden="true">/ˈsnıp.ıt/</StyledH1>
```

Comment:
Not excactly, we wanted to hide the /ˈsnɪp.ɪt/ text, because the screenreader can't read that in a understandable way. But we didn't wanted to hide the h1-semantic, so we fixed that problem with a span an moved the aria-label to the h1. Thanks for the hint!